.circleci/config.yml

```diff
@@ -331,25 +331,25 @@ jobs:
       - checkout
       - install-dependencies
       - install-dependencies:
-          directory: InstallationTests/CocoapodsInstallation
+          directory: Tests/InstallationTests/CocoapodsInstallation
       - run:
           name: Install Pods
-          working_directory: InstallationTests/CocoapodsInstallation/
+          working_directory: Tests/InstallationTests/CocoapodsInstallation/
           command: |
               pod install
       - scan-and-archive:
-          directory: InstallationTests/CocoapodsInstallation
+          directory: Tests/InstallationTests/CocoapodsInstallation
```

Comment:
@vegaro This is going to require some fixing when monorepo happens. I've create a story for this in the monorepo epic - https://revenuecats.atlassian.net/browse/CF-446

This is totally fine but just wanted to point this out 😊

Reply:
oooff didn't realize I'd break that